### PR TITLE
update ansible-lint to 6.x

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: roles/foreman/meta/main.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 dependencies:
   - role: umask
   - role: selinux
+  - role: fapolicyd
```
Comment (ekohl) [inline roles/foreman/meta/main.yml:5]: This looks unrelated

Comment (evgeni) [reply roles/foreman/meta/main.yml:5]: err, indeed. how did that land here.

**Review by ekohl — APPROVED** [review]
